src/NSKeyedUnarchiver.m

```diff
@@ -555,14 +555,22 @@ static id _decodeObjectBinary(NSKeyedUnarchiver *unarchiver, NSUInteger uid1) NS
             @throw [NSException exceptionWithName:NSInvalidUnarchiveOperationException reason:[NSString stringWithFormat:@"No classForKeyedUnarchiver class:%@", className] userInfo:nil];
             return nil;
         }
+        NSException *exception = nil;
+        if ([unarchiver requiresSecureCoding])
+        {
+            NSSet *allowed_class_list = [unarchiver allowedClasses];
```

Comment:
In objc (well, at least in Cocoa):

* use `NSThisCase` for classes (and other types) and for (typically string or integer) constants
* use `thisCase:andSo:on:` for selectors
* use `thisCase` for variables (local & instance)

Reply:
Also this is an `NSSet` so maybe it should be `allowedClassSet`, not list.

Reply:
I grepped for `allowedClassSet`, but I could not find any reference of it in the Foundation repo. 

Also, from what I understand, the [allowedClasses](https://developer.apple.com/documentation/foundation/nscoder/1412486-allowedclasses?language=objc) return an NSSet, not a list.

Reply:
That is what I'm saying, it's an `NSSet`, so you shouldn't name the variable "allowed class _list_". Maybe just `allowedClasses`, same as the method?

> I grepped for `allowedClassSet`, but I could not find any reference of it in the Foundation repo.

Sorry, I don't understand; why would there be `allowedClassesSet` if you're just adding it?

Reply:
I was being dumb. I realized that you meant rename `allowedClassList` to `allowedClassSet`.